src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using UseDApp with Unity
+description: Learn how to use UseDApp to interact with Unity engine project 
+author: "lucasespinosa28"
+tags: ["usedapp", "unity","react","dapp"]
+skill: intermediate
+lang: en
+sidebar: true
+sourceUrl: https://github.com/lucasespinosa28/usedappwithUnity
+---
+## Introduction {#introduction}
+In this tutorial, you will learn how to use [UseDApp](https://usedapp.readthedocs.io/en/latest/) a React plugins for rapid Dapp development, 
+and how to interact with a Unity using react, you will get your account balance inside a Unity and send a transaction from unity.
+
+### You need have [Unity](https://unity.com/) installed 
+
+## Unity project {#unity-project}
+On the Unity will have a text to receive the account balance, and a button to send a transaction using UseDApp,
+to communicate from unity to [UseDApp](https://usedapp.readthedocs.io/en/latest/) we will need an extra code using ```jslib```.
+
+### 1\. Create a new unity project and configure to webgl {#new-unity-project}
+Create a new unity project, the name of the project and if the project is 2d or 3d makes no difference. 
+![Captura de Tela (80)](https://user-images.githubusercontent.com/52639395/119444261-4e3a6980-bd01-11eb-9946-e7b7653918b3.png)
```

Comment:
If we could download any external images and place into the same folder (after renaming to something coherent, all kebab/lowercase), then reference using `![Alternate text explaining image](./local-image.png)`

Reply:
(Same with the others)